src/components/TermBox.vue

```diff
@@ -1,21 +1,32 @@
 <template>
-  <div class="termbox-placeholder">
-  </div>
+	<div class="termbox">
+		<div class="primary-language">{{primaryLanguageName}}</div>
+	</div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import Component from 'vue-class-component';
+import { instance as factory } from '../common/TermboxFactory';
 
-@Component
+@Component( {
+} )
 export default class TermBox extends Vue {
+
+	get primaryLanguageName(): string {
+		return factory.getLanguageNameLookup().getLanguageName( this.primaryLanguage );
+	}
+
+	get primaryLanguage() {
+		// TODO: I tried doing a mapState within the @Component annotation,
```

Comment:
https://github.com/ffxsam/vue-typescript-cookbook#im-using-vuex-mapstate-or-mapgetters-and-typescript-is-saying-the-mapped-stategetters-dont-exist-on-this ?